app/models/feedback.rb

```diff
@@ -14,7 +14,15 @@ class Feedback < ApplicationRecord
 
   validate :check_for_spam
   def check_for_spam
-    errors.add(:base, ts("^This comment looks like spam to our system, sorry! Please try again, or create an account to comment.")) unless check_for_spam?
+    errors.add(:base, ts("This report looks like spam to our system!")) unless check_for_spam?
+  end
+
+  def check_for_spam?
```

Comment:
Can you combine `check_for_spam?` and `check_for_spam`? We can avoid having 2 very similarly named methods.

Reply:
making an attempt. I'm still fairly new to Ruby so fingers crossed my solution attempt is good!